src/domain/alerts/entities/alerts.entity.ts

```diff
@@ -0,0 +1,7 @@
+export type Contract = {
+  address: string;
+  networkId: string;
```

Comment:
From a domain perspective, we have been addressing this field as `chainId` instead of `networkId`.

For consistency, I'd keep using `chainId` on the domain side and map it to `networkId` in the Tenderly API.

Reply:
Good point. I've change this in 36844eeae590fc1a6dd9d1ec0e87d8645e760b2d.